Cuahsi branding updates

aphelionz opened the pull request:
<!--

Please read, and add your text at the bottom of this message.

Thanks for contributing code to HydroShare. In order to maintain code quality and expedite this process, please assist the development team by making sure the following is present in this pull request.

For more information, see https://docs.google.com/document/d/1dzxqlZW5fKNEyQSeKiSFq-SmS-VOPCva95XXkBjPExs

-->

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

<!-- Enter steps that a QA engineer, stakeholder, or user documentation writer would follow to test the positive or "successful" case of the functionality your code provides or fixes -->

- [x] Automated Testing

<!-- Our Jenkins Instance is set up to automatically test every commit from a pull request. Code coverage must not decrease so new functionality or code paths added during a bug fix must have appropriate tests written. Every test must pass, including PEP8 code formatting tests. -->

- [x] Sufficient User and Developer Documentation

<!-- Please email your positive test case lbrazil@cuahsi.org, who will make the decision regarding user documentation. -->

- [x] Passing Jenkins Build

<!-- Our Jenkins Instance is set up to automatically test every commit from a pull request. Every test must pass, including PEP8 code formatting tests. -->

- [x] Peer Code review and approval

<!-- This is the process by which a peer developer on the HydroShare team will read the changeset, provide feedback, and ultimately give a formal approval to the code before it passes PR status. -->

### Positive Test Case

This is available for viewing at:
https://playground.hydroshare.org/

The changes are as follows:
1. Site title changed from HydroShare to CUAHSI HydroShare
2. "A CUAHSI Water Data Service" added under logo 
3. Meta title changed to "Find, Analyze and Share Water Data"
4. Changed introductory text to "HydroShare is CUAHSI's online collaboration environment for sharing data, models, and code."
5. Changed order of slideshow on home page
6. Added 5th step to "How it works": Formally Publish Data
7. Edited footer content.

hydrobuild commented:
Build finished.

hydrobuild commented:
Build finished.

aphelionz commented:
retest this, please

hydrobuild commented:
Build finished.

hydrobuild commented:
Build finished.

aphelionz commented:
This is available for viewing at:
https://playground.hydroshare.org/

The changes are as follows:
1. Site title changed from HydroShare to CUAHSI HydroShare
2. "A CUAHSI Water Data Service" added under logo 
3. Meta title changed to "Find, Analyze and Share Water Data"
4. Changed introductory text to "HydroShare is CUAHSI's online collaboration environment for sharing data, models, and code."
5. Changed order of slideshow on home page
6. Added 5th step to "How it works": Formally Publish Data
7. Edited footer content.

hyi commented on theme/templates/pages/homepage.html:
```diff
@@ -148,6 +147,21 @@ <h4>Share with colleagues</h4>
+                        </div>
+                        <div>
+                            <img class="img-responsive img-step" src="{{ STATIC_URL }}img/home-page/step1.png"
+                                 alt="Create data">
```
Step 5 is using Step1 image and tooltip which does not look right.

hyi commented on theme/templates/pages/homepage.html:
```diff
@@ -148,6 +147,21 @@ <h4>Share with colleagues</h4>
                         </div>
```
I think it is better to remove the subject ```You``` on line 142 to make it ```Choose who has access ...``` to be consistent with descriptions for other steps.

horsburgh commented:
A couple of other things I noticed testing using Chrome on my Mac:

* At some narrower browser widths, the icons under "How it works" are not shrunk symmetrically. Instead they get stretched vertically.

![image](https://user-images.githubusercontent.com/3485902/40699521-0064339c-6392-11e8-93a6-be059153ba4e.png)

* At some narrower browser widths, the text under the navigation bar "HydroShare is CUAHSI's online collaboration environment..." disappears when the navigation items wrap to the next line below the HydroShare logo.

![image](https://user-images.githubusercontent.com/3485902/40699517-f68f6a9e-6391-11e8-9071-0bca2b90c37e.png)

aphelionz commented:
Thanks Jeff! I'll look at that stuff.

aphelionz commented:
@hyi I've fixed the text changes. I couldn't update the step 5 image because I'm not a graphic designer :(

@horsburgh I fixed some spacing issues but the navbar issue you identified was there before. I changed the display of the "How it works" steps so they change to single column sooner.

aphelionz commented:
@hyi @dtarb I yanked the logo out for this round of updates, we will revisit that later. For now, everything should be up on playground for review.

hydrobuild commented:
Build finished.

dtarb commented:
In testing on playground, I noticed the following quirks in signing up.  I entered USU as my institution.  It showed red.
![image](https://user-images.githubusercontent.com/3456833/41607998-a3e32a2e-73a4-11e8-92cd-d8f46dd9bbca.png)

Then when I clicked on the email confirmation I had two institutions in my profile.
![image](https://user-images.githubusercontent.com/3456833/41608053-cc9c3852-73a4-11e8-902f-3705cb9141b3.png)
I am not sure if this is intentional.

aphelionz commented:
@dtarb Not intentional and likely a separate bug, not an artifact of this PR

dtarb commented:
On home page I see
![image](https://user-images.githubusercontent.com/3456833/41608269-57e14948-73a5-11e8-856d-cac70ffe0b48.png)
This is good.

On Discover I see
![image](https://user-images.githubusercontent.com/3456833/41608304-718b001e-73a5-11e8-8cbf-f58938f564f1.png)
I do not know if you wanted CUAHSI HydroShare here too.

On Collaborate I see Collaborate | CUAHSI HydroShare

On Apps, I get taken to the tethys portal, so am unable to see if branding at the top changed.

If these changes are being made to www, they should probably also be made to help.hydroshare.org.

In the footer I see 
![image](https://user-images.githubusercontent.com/3456833/41608710-4fb9fc14-73a6-11e8-9183-e8869a106a27.png)
This list of projects is shorter than www.  May be a manual edit.

Some of the above I think are known differences between testing and www deployments that you may be able to ignore.  I do not see any show stoppers and am OK with merging this.

+1

dtarb commented:
Perhaps not quite +1.  The problem with elongated icons that @horsburgh pointed out earlier, still persists.
![image](https://user-images.githubusercontent.com/3456833/41609040-2331a57e-73a7-11e8-9d57-4f16dc7a07f4.png)
I also think that there should be a different symbol for Formally publish than Create Data.  Same symbol for two things does not seem right.

dtarb commented:
I also suggest the following wording changes.  Under 4 on how it works.
"Choose who has access to the data and models you have uploaded to HydroShare. You can share with individual users or **make your resources public** for everyone to access."

Under 5.  I suggest "Permanently Publish"  These were words that Olin suggested.  I suggest the detailed text should be: "Obtain a Digital Object Identifier **(DOI)** so your **work** can be easily cited. Reference related journal publications in your metadata."

horsburgh commented:
Probably something like a larger version of what @Maurier used for the publish icon would be good:

![image](https://user-images.githubusercontent.com/3485902/41611417-f92216fa-73ac-11e8-94b0-63aaf5827391.png)

aphelionz commented:
@dtarb @horsburgh Happy to make the changes - do we have any graphic design resources on the team or the team's periphery that we can ask to create that graphic?

horsburgh commented:
@aphelionz - I think @Maurier did the other icons. @Maurier - do you have a few minutes to create a "publish" icon for the HydroShare front page similar to the one you used on the my resources page (pasted in my comment above)?

Maurier commented:
You  can use this one:

![publish](https://user-images.githubusercontent.com/2448568/41631093-f32cddbe-73ef-11e8-9a21-f6ed9b7aeeb1.png)

And here is the .psd file I used to generate them:

[HS ICONS.zip](https://github.com/hydroshare/hydroshare/files/2117352/HS.ICONS.zip)

aphelionz commented:
@Maurier Thank you!

@dtarb and @horsburgh Updated this. It should be up on playground shortly. Note that the title changes require a local_settings.py fix so that will be a deployment task.

hydrobuild commented:
Build finished.

horsburgh commented:
Looks good to me now on playground.

dtarb commented:
The wording suggestions I made above for changes to 4 and 5 under how it works have not been done.  I am OK if you are deliberately not taking these suggestions.  But if overlooked, please fix.

horsburgh commented:
@aphelionz - I think @dtarb 's suggested working makes more sense now that there are 5 points instead of 4. The 4th point shouldn't talk about publishing.

aphelionz commented:
@dtarb @horsburgh All set and should be up on playground shortly.

hydrobuild commented:
Build finished.

dtarb commented:
Playground still says "Formally Publish"  It should say "Permanently Publish".  

I did also notice that what was formerly "Discover | HydroShare" has changed to "Discovery | CUAHSI HydroShare".  I think it should be "Discover | CUAHSI HydroShare" to be consistent with the menu item and active rather than passive voice.
![image](https://user-images.githubusercontent.com/3456833/42464108-4ce00e4c-8365-11e8-90c5-4415c3d4de20.png)
 
That said - if the switch to Discovery is deliberate motivated by some rationale - I am OK with it.  If it was a slip, then please use "Discover"

dtarb commented:
@hyi welcome back.  Please review the code, so that when @aphelionz implements the content changes I have commented on this can be merged.

hyi commented:
@dtarb Will do. I just did a quick check on playground and did find the wording is indeed changed to ```When **permanently** published, this resource will have a formal ...``` unless you meant some other place. I'll defer to @aphelionz whether he would change ```Discovery``` to ```Discover``` as you indicated above. Then I can review the code and approve the PR accordingly.

dtarb commented:
The word I am asking be changed is
![image](https://user-images.githubusercontent.com/3456833/42465805-44d43b06-836a-11e8-8c0c-de00f3089c89.png)

hyi commented:
@dtarb Got it. Thanks. I'll review the code after @aphelionz make these further changes.

aphelionz commented:
@dtarb @hyi changes have been made and are building on playground now. You may review when ready.

hydrobuild commented:
Build finished.

aphelionz commented:
@dtarb Maybe you want to take one last look?

dtarb commented:
@aphelionz playground is not responding.  Let me know when you want me to take the last look.
